server/README.md

```diff
@@ -171,5 +171,5 @@ curl -X DELETE http://localhost:5000/api/message/0
 
 Get all messages for one RoomId:
 ```sh
-curl -v http:/localhost:5000/api/message?roomid=6043978fd68dc3fcbf6079d6 # doesn't work here but it doesn't worked before need to open an issue
+curl -v http:/localhost:5000/api/message?roomId=6043978fd68dc3fcbf6079d6 # doesn't work here but it doesn't worked before need to open an issue
```

Comment:
Remove the comment `# doesn't work here but it doesn't worked before need to open an issue`